packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -178,10 +178,13 @@ class DependencyExtractionWebpackPlugin {
 					continue;
 				}
 
-				const assetFilename = buildFilename.replace(
-					/\.js$/i,
-					'.asset.' + ( outputFormat === 'php' ? 'php' : 'json' )
-				);
+				// We assume that anything after the first period is
+				// the extension. This is in order to ignore suffixes
+				// like .min.js, for example.
+				const assetFilename =
+					buildFilename.split( '.' )[ 0 ] +
+					'.asset.' +
+					( outputFormat === 'php' ? 'php' : 'json' );
```

Comment:
This doesn't work because the `buildFilename` starts with period: `./build/a11y/index.min.js`. I fixed this in 6bcdfc2ae4 with a regexp replacement.

Reply:
Thanks 👍🏻